src/optimize/dynamic_dll_plugin/dll_config_model.js

```diff
@@ -86,19 +86,13 @@ function generateDLL(config) {
           // Self calling function with the equivalent logic
           // from maybeAddCacheLoader one from base optimizer
           use: ((babelLoaderCacheDirPath, loaders) => {
-            // Only deactivate cache-loader and thread-loader on
-            // distributable. It is valid when running from source
-            // both with dev or prod bundles or even when running
-            // kibana for dev only.
-            if (IS_KIBANA_DISTRIBUTABLE) {
-              return loaders;
-            }
-
             return [
               {
                 loader: 'cache-loader',
                 options: {
-                  cacheDirectory: babelLoaderCacheDirPath
+                  cacheContext: fromRoot('.'),
+                  cacheDirectory: babelLoaderCacheDirPath,
+                  readOnly: !!IS_KIBANA_DISTRIBUTABLE && !(process && process.env && process.env.FORCE_WRITABLE_CACHE)
```

Comment:
I think the double-negative of `FORCE_WRITIBLE_CACHE` would be a little easier to grok if it was written as:

```js
process.env.FORCE_WRITABLE_CACHE ? false : IS_KIBANA_DISTRIBUTABLE
```

Reply:
👍